api/v1/_uploadPublicFile/PKPUploadPublicFileHandler.inc.php

```diff
@@ -134,7 +134,7 @@ public function uploadFile($slimRequest, $response, $args) {
 				)
 			)
 		);
-		$extension = end(explode('.', strtolower(trim($filename))));
+        $extension = pathinfo(strtolower(trim($filename)), PATHINFO_EXTENSION);
```

Comment:
This accidentally introduces spaces for indentation, rather than tabs; could you tweak it back to tabs?

Reply:
Changed spaces to tabs as requested